Document cagg improvements (#1529)
timescale/docs: timescaledb/how-to-guides/continuous-aggregates/create-index.md

```

## Manually create and drop indexes

You can use a regular PostgreSQL statement to create or drop an index on a
continuous aggregate. For example, to create an index on `avg_temp` for a
materialized hypertable named `weather_daily`:

> 

Reviewer: Would be nice if we have a note here about `DROP INDEX` because to work the user should use the internal schema `_timescaledb_internal.index_name` because this schema is where the materialization hypertable is stored. So, for example:

```sql
CREATE TABLE conditions (
       time TIMESTAMPTZ NOT NULL,
       location TEXT NOT NULL,
       temperature DOUBLE PRECISION NULL
);

SELECT create_hypertable('conditions', 'time');

CREATE MATERIALIZED VIEW conditions_daily
WITH (timescaledb.continuous, timescaledb.materialized_only = false) AS
SELECT location,
       time_bucket(INTERVAL '1 day', time) AS bucket,
       AVG(temperature)
  FROM conditions
GROUP BY location, bucket
WITH NO DATA;

CREATE INDEX index_avg ON conditions_daily (avg);

DROP INDEX _timescaledb_internal.index_avg;
```

```sql

> CREATE INDEX avg_temp_idx ON weather_daily (avg_temp);

Reviewer: ```suggestion
CREATE INDEX avg_temp_idx ON weather_daily (avg_temp);
DROP INDEX _timescaledb_internal.avg_temp;
```

```

### Limitations on created indexes

In TimescaleDB 2.7 and above, you can create an index on any column in the
materialized view. This includes aggregated columns, such as those storing sums
and averages. In earlier versions of TimescaleDB, you can't create an index on
an aggregated column.

> In all versions of TimescaleDB, you can't create a unique index on a continuous

Reviewer: ```suggestion
In all versions of TimescaleDB, you can't create unique indexes on a continuous
```

> In all versions of TimescaleDB, you can't create a unique index on a continuous
> aggregate.

Reviewer 2: ```suggestion
You can't create a unique index on a continuous aggregate, in any version of TimescaleDB.
```
